hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -214,12 +229,15 @@ public static void main(final String... args) throws Exception {
                     "Multiple nodes were supplied via the command line. Only one node can be started per java process.");
             exitSystem(NODE_ADDRESS_MISMATCH);
         }
-        final List<NodeId> nodesToRun = getNodesToRun(diskAddressBook, commandLineArgs.localNodesToStart());
-        checkNodesToRun(nodesToRun);
-        final var selfId = ensureSingleNode(nodesToRun, commandLineArgs.localNodesToStart());
         final var platformConfig = buildPlatformConfig();
+        final var cliNodesToRun = commandLineArgs.localNodesToStart();
+        final var envNodesToRun = platformConfig.getConfigData(BasicConfig.class).nodesToRun().stream()
```

Comment:
`envNodesToRun` is a little "short" `configNodesToRun` would fit better since the the value could possible come from a config file as well

Reply:
makes sense.